internal/engine/eval/vulncheck/review.go

```diff
@@ -32,57 +29,10 @@ import (
 )
 
 const (
-	reviewBodyMagicComment = "<!-- minder: pr-review-body -->"
-	commitStatusContext    = "minder.stacklok.dev/pr-vulncheck"
-	vulnsFoundText         = `
-Minder found vulnerable dependencies in this PR. Either push an updated
-version or accept the proposed changes. Note that accepting the changes will
-include Minder as a co-author of this PR.
-`
-	vulnsFoundTextShort = `
-Vulnerable dependencies found.
-`
-	noVulsFoundText = `
-Minder analyzed this PR and found no vulnerable dependencies.
-`
-	reviewBodyDismissCommentText = `
-Previous Minder review was dismissed because the PR was updated.
-`
-	vulnFoundWithNoPatch = "Vulnerability found, but no patched version exists yet."
+	stacklokBotUser     = "stacklokbot"
```

Comment:
Hmm, I don't think we want to hardcode a bot username here. Later the code does:
```
		isMinder := review.GetUser().GetLogin() == stacklokBotUser && strings.HasPrefix(review.GetBody(), reviewBodyMagicComment)
```
Could we instead use:
```
		isMinder := review.GetUser().GetLogin() == ghCli.GetAuthenticatedUser() && strings.HasPrefix(review.GetBody(), reviewBodyMagicComment)
```

Reply:
It would return the user who enrolled that particular org or personal account, so in the case of stacklok's deployment, it would be stacklokbot.

Reply:
Done!